src/test/java/com/hotels/mutantswarm/model/MutantSwarmScriptTest.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.mutantswarm.model;
+
+import static java.util.Collections.singletonList;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.antlr.runtime.CommonToken;
+import org.antlr.runtime.TokenRewriteStream;
+import org.apache.hadoop.hive.ql.parse.ASTNode;
+import org.apache.hadoop.hive.ql.parse.MutantSwarmParseDriver;
+import org.apache.hadoop.hive.ql.parse.ParseException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.hotels.mutantswarm.model.MutantSwarmStatement.Factory;
+
+@RunWith(MockitoJUnitRunner.class)
+public class MutantSwarmScriptTest {
+  @Mock
+  private MutantSwarmParseDriver parseDriver;
+  @Mock
+  private ASTNode tree;
+  @Mock
+  private TokenRewriteStream tokenStream;
+  @Mock
+  private CommonToken token;
+
+  private MutantSwarmStatement statement;
+  private List<MutantSwarmStatement> statements;
+  private MutantSwarmScript.Impl script;
+  private Factory factory;
+
+
+  @Before
+  public void setupMocks() throws ParseException{
+    factory = new MutantSwarmStatement.Factory(parseDriver);
+    when(parseDriver.lex("SELECT * FROM x WHERE a = 1")).thenReturn(tokenStream);
+    when(parseDriver.parse(tokenStream)).thenReturn(tree);
+    when(parseDriver.extractTokens(tokenStream)).thenReturn(singletonList(token));
+    statements = new ArrayList<MutantSwarmStatement>();
+    statement = factory.newInstance(0, 1, "SELECT * FROM x WHERE a = 1");
+    statements.add(statement);
+    script = new MutantSwarmScript.Impl(0,Paths.get("/Users/user/eclipse-workspace/mutant-swarm/target/test-classes/mutantSwarmTest/scriptToTest1.sql"),statements);
```

Comment:
This path won't work on other people's machines... if you don't actually need this to be a valid path make it something clearer like "/some/path/to/script.sql"